app/javascript/packs/searchParams.js

```diff
@@ -1,6 +1,10 @@
 /* global checkUserLoggedIn, showLoginModal, userData, buildArticleHTML, initializeReadingListIcons */
 /* eslint no-undef: "error" */
 
+// This is a lightweight version of the client, which we should be fine importing regardless of whether it is set up.
+// Could be optimized for optional inclusion in the future.
+const algoliasearch = require('algoliasearch/lite'); 
```

Comment:
non-blocking thought: Weird that their doc suggest this method, the common way to do this is
`import algoliasearch from 'algoliasearch/lite'` This is untested, though.